GDJS/Runtime/variable.js

```diff
@@ -45,7 +60,8 @@ gdjs.Variable = function(varData)
 
 			if (varData.children !== undefined) {
 	        	for(var i = 0, len = varData.children.length;i<len;++i) {
-			    	var childData = varData.children[i];
+					var childData = varData.children[i];
+					/** @type {gdjs.Variable} */
```

Comment:
You might not need this type? `new gdjs.Variable` will make Typescript understand that you created a gdjs.Variable I think.

Reply:
I like to type everything. Wihout that at least VSCode don't recognize the type so I think it's better to include that